README.md

```diff
@@ -157,14 +175,77 @@ kube-prometheus-stack-prometheus-node-exporter-zsls7       1/1     Running   0
 prometheus-kube-prometheus-stack-prometheus-0              2/2     Running   1          8m21s
 prometheus-kube-prometheus-stack-prometheus-1              2/2     Running   1          8m21s
 prometheus-kube-prometheus-stack-prometheus-2              2/2     Running   1          8m21s
+
+➜ kubectl -n graylog get pods
+NAME                           READY   STATUS      RESTARTS   AGE
+elasticsearch-master-0         1/1     Running     0          8m21s
+elasticsearch-master-1         1/1     Running     0          8m21s
+elasticsearch-master-2         1/1     Running     0          8m21s
+fluentd-6jmhn                  1/1     Running     0          8m21s
+fluentd-9lc2g                  1/1     Running     0          8m21s
+fluentd-cfzqv                  1/1     Running     0          8m21s
+fluentd-g88cr                  1/1     Running     0          8m21s
+fluentd-ks5zx                  1/1     Running     0          8m21s
+fluentd-mdg8p                  1/1     Running     0          8m21s
+fluentd-qnn8b                  1/1     Running     0          8m21s
+fluentd-x4vjd                  1/1     Running     0          8m21s
+fluentd-zwzfw                  1/1     Running     0          8m21s
+graylog-0                      1/1     Running     0          8m21s
+graylog-1                      1/1     Running     0          8m21s
+mongodb-mongodb-replicaset-0   2/2     Running     0          8m21s
+mongodb-mongodb-replicaset-1   2/2     Running     0          8m21s
+mongodb-mongodb-replicaset-2   2/2     Running     0          8m21s
+
 ➜ kubectl -n cert-manager get pods
 NAME                            READY   STATUS    RESTARTS   AGE
-cert-manager-776cd4f499-688bm   1/1     Running   0          1h
+cert-manager-77bbfd565c-rf7wh              1/1     Running   0          8m21s
+cert-manager-cainjector-75b6bc7b8b-dv2js   1/1     Running   0          8m21s
+cert-manager-webhook-8444c4bc77-jhzgb      1/1     Running   0          8m21s
+```
+
+Other ways to ensure that installation have been successful is to check application logs for errors and exceptions. 
+
+#### Ensure Kafka cluster is functional and RADAR-base topics are loaded
+
+```bash
+➜  kubectl exec -it cp-kafka-0 -c cp-kafka-broker -- kafka-topics --zookeeper cp-zookeeper-headless:2181 --list | wc -l
+273
+```
+This output means there are 273 topics loaded in the Kafka cluster.
+In your setup, the number of topics can be more or less, depending on components that you have activated.
+
+#### Troubleshoot
```

Comment:
Might be useful to on the wiki compile some common issues (I'm sure we have a few common issues that we encounter or people will report soon). Also handy to have a list of things to send people to work through initially, broadly the more we can avoid bespoke support the better.